docs/tutorials/config.md

```diff
@@ -383,14 +383,14 @@ evaluation = dict(pipeline=[  # Pipeline is passed by eval_pipeline created befo
 ])
 lr = 0.008  # Learning rate of optimizers
 optimizer = dict(  # Config used to build optimizer, support all the optimizers in PyTorch whose arguments are also the same as those in PyTorch
-    type='Adam',  # Type of optimizers,   # Type of optimizers, refer to https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/optimizer/default_constructor.py#L13 for more details
+    type='Adam',  # Type of optimizers,   # Type of optimizers, refer to https://github.com/open-mmlab/mmcv/blob/v1.3.7/mmcv/runner/optimizer/default_constructor.py#L12 for more details
```

Comment:
Updating the urls with fixed codebase version to avoid the line linking confusion caused by continually updated master branch.